src/psbt/finalizer.rs

```diff
@@ -252,7 +252,9 @@ pub fn finalize<C: secp256k1::Verification>(
     for (n, input) in psbt.inputs.iter().enumerate() {
         let target = input.sighash_type.unwrap_or(bitcoin::SigHashType::All);
         for (key, rawsig) in &input.partial_sigs {
-            if rawsig.is_empty() {
+            // If it comports a SIGHASH flag, it must at least be 71-bytes long.
+            // If it doesn't, it's invalid. FIXME: it's only valid for ECDSA sigs.
+            if rawsig.len() < 71 {
```

Comment:
I think it is also possible to have short signatures in bitcoin. I think it is possible to have shorter signatures than 71 bytes. ECDSA.  
https://github.com/bitcoin/bitcoin/blob/ea5a50f92a6ff81b1d2dd67cdc3663e0e66733ac/src/secp256k1/src/ecdsa_impl.h#L189-L190

I think with a probability of 1/1024 we can have sigs of 70 bytes. 1/(1024*256) 69 bytes and so on. Infact, Rusell from Blockstream has used sigs as short as `300602010102010101`   on testnet.

Reply:
It's valid by consensus but i'd have thought it's non standard ? Would have to check though.

Reply:
I don't think there should be a standardness rule against it. 70 bytes sigs can occur naturally with a 0.1% chance. I could not find one in bitcoin core `policy.h`, but I have had cases something was not included `policy.h` .

Reply:
There is no standardness rule about the length of signatures. Why would it be illegal to try to save some bytes?

Without parsing DER it is impossible to tell what the "real" length of an ECDSA signature is.